prep-javascript-variables/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Raymond Ho';
+    console.log('Full Name:', fullName);
+    var numberOfPets = 2;
+    console.log('Number of Pets:', numberOfPets);
+    var likesVegetables = true;
+    console.log('Likes Vegetables:', likesVegetables);
+    var nothing = 'null';
+    console.log('Nothing:', nothing);
```

Comment:
We want to leave the quotes off of `null` so that JavaScript doesn't interpret it as a string.

Reply:
<img width="457" alt="Screen Shot 2022-09-27 at 4 12 27 PM" src="https://user-images.githubusercontent.com/113560956/192653596-d03505b7-2781-4118-b99f-823f1b995a3a.png">

Deleted quotation marks from null.